Code/Engine/Foundation/ezEngine.natvis

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
 	<Type Name="vk::Flags&lt;*&gt;">
-		<DisplayString>[{($T1)m_mask}]</DisplayString>
+		<DisplayString>[{($T1)m_uiMask}]</DisplayString>
```

Comment:
Note that this visualizer is for the vulkan flags type and there the member is called m_mask.

Reply:
Ah sorry, search an replace went overboard. Thanks for noticing. I think I'll make a natvis zoo to test these.